consumer.go

```diff
@@ -234,6 +234,7 @@ func (c *Consumer) ScanShard(ctx context.Context, shardID string, fn ScanFunc) e
 var retriableErrors = map[string]struct{}{
 	kinesis.ErrCodeExpiredIteratorException:               struct{}{},
 	kinesis.ErrCodeProvisionedThroughputExceededException: struct{}{},
+	kinesis.ErrCodeInternalFailureException:               struct{}{},
```

Comment:
I think this has a risk of being overly broad and causing infinite error loops but InternalFailure errors _should_ be retriable IMO

Reply:
```
// ErrCodeInternalFailureException for service response error code
    // "InternalFailureException".
    //
    // The processing of the request failed because of an unknown error, exception,
    // or failure.
```

I haven't seen this error before; but agree it feels like it _should_ be retriable